blueman/DeviceClass.py

```diff
@@ -97,9 +97,9 @@
     # translators: device class
     _("Set-top box"),
     # translators: device class
-    _("Hifi audio"),
+    _("Hi-Fi audio"),
```

Comment:
https://btprodspecificationrefs.blob.core.windows.net/assigned-numbers/Assigned%20Number%20Types/Baseband.pdf says `HiFi`. I'd stick to that.

Reply:
Pedantic person on the Internet says:
"HiFi" is the worst of all possible options. It is neither "Hi-fi" or "Hifi", but it is also the worst option of the two.
Moreover if you wanted to use big F, use "Hi-Fi", from DIN 45500.
http://brakken.no/el2/avgrensa/din45500.pdf

I am also an audio guy, and "HiFi" just looks, wrong.